lib/tasks/stateoscope.rake

```diff
@@ -1,8 +1,10 @@
 namespace :stateoscope do
   desc 'visualize state machine for a given class'
-  task :visualize, [:class, :state_machine_name] => :environment do |t, args|
-    fail ArgumentError, 'missing required argument <class>' unless args[:class].present?
+  task :visualize, %i[class state_machine_name current_state] => :environment do |_, args|
+    raise ArgumentError, 'missing required argument <class>' if args[:class].nil?
```

Comment:
`if x.nil?` is not exactly the same as `unless x.present?` - it now allows blank strings. I would prefer that a blank string still trigger the exception here. I think `if x.blank?` will do the trick.

Reply:
Yes sorry not sure why I changed that. I'm working with vscode and a rubocop extension and probably it complained about that part because `unless something` can always be rewritten with `if the opposite`. Sure `nil?` is not the opposite of `present?` I know. 

By the way `blank?`, `present?`, `classify`, `constantize` are all Rails. Have you tried to use the gem outside of Rails?

Reply:
Yes I thought about that as well while writing this comment but at the moment there is a dependency on activesupport in the gemfile anyway. We might want to get rid of that in the future, see #7